artifacts/bump.sh/train-travel/1.0/train-travel.1.0.api.yml

```diff
@@ -0,0 +1,57 @@
+    point. It's a sample OpenAPI description for an imaginary Pet Store with an
+    API, but the OpenAPI is old, and the API it describes is pretty far from
+    best practices. We thought it was time for a refresh, so we're bringing you
+    the Train Travel API, a new sample OpenAPI you can use for your tooling and
+    testing. 
+    
+    ### Introducing the Train Travel API
+
+    The OpenAPI description document is on the train-travel-api GitHub
+    repository, and comes in the form of a single openapi.yaml that you can use
+    as a sample for any documentation, validation, mocking, or whatever tools
+    that you maintain and want to show a working demo with something less
+    contrived than a "Todo API" or the Pet Store.
+
+    It's an open-source project licensed as Creative Commons
+    Attribution-NonCommercial-ShareAlike 4.0 International.
+    
+    ### Describes a Realistic API
+
+    This API builds off of various open data sources and public APIs that have
+    all proven the concepts and patterns used in the design of this API.
+
+    The concept of Stations is based on [Stations - A Database of European Train
+    Stations](https://github.com/trainline-eu/stations), maintained by
+    [Trainline EU](https://github.com/trainline-eu/), and powered by
+    [OpenStreetMap](https://www.openstreetmap.org/), [SNCF
+    OpenData](https://ressources.data.sncf.com/explore/dataset/referentiel-gares-voyageurs/),
+    [Digitraffic.fi](http://rata.digitraffic.fi/api/v1/metadata/stations),
+    [OpenTransportData.swiss](https://opentransportdata.swiss/en/dataset/bhlist/resource/035c4d98-df6c-49b4-988a-3822823f0bf9),
+    [admin.ch](https://www.bav.admin.ch/bav/fr/home/themes-a-z/geoinformation/geodonnees-de-base/arrets-des-transports-publics.html).
+
+  icon:
+    - base64data: iVBORw0KGgoAAAANSUhEUgAAADAAAAAwCAIAAADYYG7QAAACZUlEQVR4nOzXy2sTQRwH8NlH0hLRutYeugQiHgJBcIN4SAJpbhVqwP/AgwfBQwVrG9GKUMGTioo3e1FvPg4BKxjIyRwitehBMSJUSGzDmlAHa2v3kZ0R2uBmQ+wMtRktzPeSZPa3mU92Jvlt5P79A+B/ivivAe3hIFI4iBQOIoWDSOEgUmRihaZpx4aH2wYbTqNU+pjP523b7ngW3jOIDyZRzz7w7YtUKQjm8raBotFoJjPR8VC1Wp3IXMjlcp5RQQQjF/HRk8gCjmGjtYa5suKbvev/9IgG9FdLpqrqwwf347GYx3PiChg6DcT1j4rXh+Re48g5M5TuOggAIEmS5/oNHBBTp1p07lPj0Bks+rsOAgAkEoldgUATEB0BouQewy4L+/vs/igLkCRJexWlCdq9ye0exn6FBcg0zXq93pxzqdJJsvEgiGs6C9CLXM6yrObUb59j82cLomWm1QUZvu86CEI4NXXVfb1cQ08nAfZyBAAcK/DmmoCd7oJez80dT6crFe8yzT4GT8aB+cNdqlU9UBiXl97RvKdA/NcRCoU0TWsbRMiZn/9cKpX+dBaWe7F6GPf0Cd91of6B5trQghhnBzbXcDjc1hzo4yC0uLhQLL4yDGPbQPF4/OaN61sDbQRCODZ2/tnMDE0xiyVTFGV6+l4qlaIpZrSHZFm+c/uWz+cjVrLb1MFgMEaxF5l+y1R1kFjDFKTrX4k17EC1Wq1YLBLL2IEuTV7+fVOwSViAIISjo2ez2SxNMbmXRSKRoWRyaxSEULlcflko0P9S8+ZKCgeRwkGkcBApHEQKB5HyKwAA//8dEMgpImCtowAAAABJRU5ErkJggg==
+      mediatype: image/png
+spec:
+  version: '1.0'
+  capabilities: Full Mocks
+  contracts:
+    - type: openapi
+      url: https://raw.githubusercontent.com/bump-sh-examples/train-travel-api/main/openapi.yaml
```

Comment:
It looks like this OpenAPI file does not contain examples that can be imported directly into Microcks. Is it the correct URL? I think you were working on a different branch...

Reply:
Got it! The OpenAPI lives on another branch and the URL should be https://raw.githubusercontent.com/bump-sh-examples/train-travel-api/refs/heads/feat/microcks/openapi.yaml instead. Be ware that https://docs.bump.sh/guides/bump-sh-tutorials/mocking-with-microcks/ is mentioning the OpenAPI file on `main` branch too!

Reply:
@lbroudoux thanks for prompting me to get on with it. I have a PR open to get the changes merged, which I've just tidied up ready for review. It's got a few questions on there which you might be interested in. https://github.com/bump-sh-examples/train-travel-api/pull/21